src/advanced/node-management.md

```diff
@@ -516,40 +512,41 @@ the HTTP status of the response is set to 404.
 
 ##### Known Uncommitted Transaction
 
-Response JSON has same fields as
-[`SerializedTransaction`](#serializedtransaction)
-plus `type` field with value equal to `"in-pool"`.
+Response JSON has the same fields as
+[`Content`](#content) plus `type` field with value equal to `"in-pool"`.
 
+<!-- markdownlint-disable MD013 -->
 ??? example "Response Example"
     ```json
     {
-      "type": "in-pool",
-      "body": {
-        "amount": "152",
-        "from": "b0d6af8bbe45c574c5f9dd8876b5b037b38d1bf861fd7b90744957aa608ed0c2",
-        "seed": "2953135335240383704",
-        "to": "99e396355cb2146aba0457a954ebdae36e09e3abe152693cfd1b9a0975850789"
-      },
-      "protocol_version": 0,
-      "service_id": 128,
-      "message_id": 128,
-      "signature": "7d3c503d6dc02ca24faaeb37af227f060d0bcf5f40399fae7831eb68921fd00407f7845affbd234f352d9f1541d7e4c17b4cd47ec3f3208f166ec9392abd4d00"
+    "type": "in-pool",
+    "content": {
+      "debug": {
+        "to": {
+          "data": [242, 39, 156, 45, 245, 89, 252, 58, 143, 174, 88, 196, 76, 225, 223, 206, 177, 125, 63, 3, 90, 36, 128, 213, 70, 37, 243, 231, 174, 139, 81, 151]
+          },
```

Comment:
Nit: formatting here looks weird. This closing brace should be aligned with `"to":`, for example.